sycl/doc/extensions/experimental/sycl_ext_oneapi_work_group_static.asciidoc

```diff
@@ -58,27 +52,26 @@ not rely on APIs defined in this specification.*
 
 == Overview
 
-This extension adds a way to allocate device local memory, without passing a
-kernel argument.
+This extension adds a ways to allocate device local memory, without passing a
```

Comment:
Either "a way" or "ways".  Seems like you want singular here.

Reply:
fixed, thanks